src/basic/REDUCE3_INT-OMPTarget.cpp

```diff
@@ -62,31 +62,74 @@ void REDUCE3_INT::runOpenMPTargetVariant(VariantID vid, size_t RAJAPERF_UNUSED_A
 
   } else if ( vid == RAJA_OpenMPTarget ) {
 
-    startTimer();
-    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+    if (tune_idx == 0) {
+
+      startTimer();
+      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-      RAJA::ReduceSum<RAJA::omp_target_reduce, Int_type> vsum(m_vsum_init);
-      RAJA::ReduceMin<RAJA::omp_target_reduce, Int_type> vmin(m_vmin_init);
-      RAJA::ReduceMax<RAJA::omp_target_reduce, Int_type> vmax(m_vmax_init);
+        Int_type tvsum = m_vsum_init;
```

Comment:
No more old reducers?

Reply:
Crap. That's a cut-paste error. I will fix.  

Although I can't find a compiler that produced correct results for the old reduction interface with OpenMP Target. So we may want to get rid of it -- easy transition since no one uses it.

Reply:
ya, sounds reasonable to me, should we remove the ompt old reduction impl from RAJA at some point?

Reply:
Code fix pushed. 

I think we can remove it at some point while we are cleaning up other things. Always easier to maintain less code.